tests/commonTemplates_test.go

```diff
@@ -75,6 +78,39 @@ var _ = Describe("Common templates", func() {
 			err := apiClient.Get(ctx, testTemplate.GetKey(), testTemplate.NewResource())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("[test_id:5352]creates only one default variant per OS", func() {
```

Comment:
test_id: 5545 (test id 5352 is verify that only one template per os-flavor-workload has "template.kubevirt.io/default-os-variant=true")

Reply:
Then this should be right, no?
5545 should be https://github.com/kubevirt/ssp-operator/pull/62

Reply:
sorry; was looking at the worng pr